### PR TITLE
Add Oracledb module details to standard library repo

### PR DESCRIPTION
**Pull request by lahirusamith**
$Subject as OracleDB module is ready to publish to ballerina central.

Fixes ballerina-platform/ballerina-standard-library#1894

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,3 +76,4 @@ Ballerina code is distributed under [Apache license 2.0](https://github.com/ball
...
...inax-oracledb/releases)| [![Build](https://github.com/ballerina-platform/module-ballerinax-oracledb/actions/workflows/build-timestamped-master.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-oracledb/actions/workflows/build-timestamped-master.yml)| [![Trivy](https://github.com/ballerina-platform/module-ballerinax-oracledb/actions/workflows/trivy-scan.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-oracledb/actions/workflows/trivy-scan.yml)| [![CodeCov](https://codecov.io/gh/ballerina-platform/module-ballerinax-oracledb/branch/main/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/module-ballerinax-oracledb)| [![Bugs](https://img.shields.io/github/issues-search/ballerina-platform/ballerina-standard-library?query=is%3Aopen+label%3AType%2FBug+label%3Amodule%2Foracledb&label=&color=brightgreen&logo=github)](https://github.com/ballerina-platform//ballerina-standard-library/issues?q=is%3Aopen+label%3AType%2FBug+label%3Amodule%2Foracledb)| [![GitHub pull-requests](https://img.shields.io/github/issues-pr/ballerina-platform/module-ballerinax-oracledb.svg?label=)](https://github.com/ballerina-platform/module-ballerinax-oracledb/pulls)|
```
Comment (niveathika) [inline README.md:79]: This is autogenerated no need to edit here

File: release/resources/stdlib_modules.json [hunk 2]
```diff
@@ -401,7 +401,8 @@
                 "module-ballerinax-java.jdbc",
                 "module-ballerinax-mssql",
                 "module-ballerinax-mysql",
-                "module-ballerinax-postgresql"
+                "module-ballerinax-postgresql",
```
Comment (niveathika) [inline release/resources/stdlib_modules.json:404]: Same

File: README.md (hunk 1 above)
Comment (lahirusamith) [reply README.md]: removed in https://github.com/ballerina-platform/ballerina-standard-library/pull/1929/commits/3d92859879abd09244544e8666289860f78170e5

File: release/resources/stdlib_modules.json (hunk 2 above)
Comment (lahirusamith) [reply release/resources/stdlib_modules.json]: removed in https://github.com/ballerina-platform/ballerina-standard-library/pull/1929/commits/3d92859879abd09244544e8666289860f78170e5

**Review by niveathika — APPROVED** [review]

**Pull request merged**